packages/sanity/src/core/releases/components/BundleMenuButton/__tests__/BundleMenuButton.test.tsx

```diff
@@ -116,7 +116,7 @@ describe('BundleMenuButton', () => {
     screen.getByText('This will also delete 2 document versions', {exact: false})
 
     await act(() => {
-      fireEvent.click(screen.getByText('Confirm'))
+      fireEvent.click(screen.getByText('Delete'))
```

Comment:
This shouldn't be an issue - when we come to localise, we are already wrapping our test components in a `createTestWrappper` which will make sure that all the `t('releases.delete')` will correctly appear in the tests as 'Delete'. 

See here for where we are already passing the locale bundle through, it's just empty right now - https://github.com/sanity-io/sanity/pull/7159/files/9af485a8b930519df05cda5332ff2c5896cb3ac6#diff-30b5cd035e5bb3d42940ef843825f7a1040ed933a6d400c9b297176105fdf2e7R26

Reply:
Great! Apologies, I'm not familiar with how we handle localisation in tests.

It does feel potentially problematic to select by text rather than some other selector (test id, role, etc) e.g. in case the copy changes. But I've also noticed with some Sanity UI components, there's no way for us to add such an attribute.

I don't think it's worth blocking. Thanks for explaining! 🙂